.github/workflows/ci.yml

```diff
@@ -14,20 +14,21 @@ jobs:
       fail-fast: false
       matrix:
         container_image:
-          - quay.io/centos/centos:stream8
           - quay.io/centos/centos:stream9
-          - registry.fedoraproject.org/fedora:37
-          - registry.fedoraproject.org/fedora:38
           - registry.fedoraproject.org/fedora:39
+          - registry.fedoraproject.org/fedora:40
           - registry.fedoraproject.org/fedora:rawhide
           - registry.access.redhat.com/ubi8
           - registry.access.redhat.com/ubi9
         dotnet_version:
           - "6.0"
-          - "7.0"
+          - "8.0"
+        exclude:
+          - container_image: registry.fedoraproject.org/fedora:rawhide
+            dotnet_version: "6.0"
         include:
-          - container_image: quay.io/centos/centos:stream9
-            dotnet_version: "8.0"
+          - container_image: registry.fedoraproject.org/fedora:40
```

Comment:
Could you add `registry.fedoraproject.org/fedora:rawhide` ? Or replace fedora:40 with that?

Reply:
I'll add it.